common/properties/coverageLevels.txt

```diff
@@ -95,6 +97,7 @@ pl ;	modern
 ps ;	modern
 pt ;	modern
 qu ;	moderate
+raj ;	basic
```

Comment:
I wonder if this is related to https://unicode-org.atlassian.net/browse/CLDR-15954